src/app/containers/ATIAnalytics/params/cpsAssetPage/buildParams.test.js

```diff
@@ -49,6 +61,24 @@ const expectation = {
     '31684f19-84d6-41f6-b033-7ae08098572a~3d5d5e30-dd50-4041-96d5-c970b20005b9',
 };
 
+const expectationLegacy = {
+  appName: serviceContext.atiAnalyticsAppName,
+  categoryName: undefined,
```

Comment:
Do we need to define these as undefined?  Can they just be deleted?

Reply:
Yessir